utils/mil_tools/nodes/tf_fudger.py

```diff
@@ -122,11 +122,12 @@ def reset():
     if k == ord('z'):
         set_bars((0, 0, 0), (0, 0, 0))
 
-    if k == ord('s'):
+    if False and k == ord('s'):
```

Comment:
This part of the script may be removed, else you'd have to specify the path for the launch file